docs/reference/form_types.rst

```diff
@@ -38,8 +38,10 @@ The service definition should contain the following arguments:
 
 .. warning::
 
-    ``DoctrineORMSerializationType`` cannot be used directly with Symfony3.0, you need to extends the class ``BaseDoctrineORMSerializationType``
-    with an empty class to have an unique FQCN.
+    ``DoctrineORMSerializationType`` cannot be used directly with
```

Comment:
This is deprecated and removed on master, maybe remove it from docs?

Reply:
Yes, but in another PR